srcpkgs/python-xcffib/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'python-xcffib'
+pkgname=python-xcffib
+version=0.8.1
+revision=1
+archs=nocross
```

Comment:
I don't think this is correct because it now fail with this
```
$ ./xbps-src pkg qtile
=> qtile-0.13.0_1: removing autodeps, please wait...
=> qtile-0.13.0_1: building [python3-module] ...
   [host] python3-setuptools-41.2.0_2: found (https://alpha.de.repo.voidlinux.org/current)
   [host] python3-cairocffi-1.1.0_1: not found
   [host] python3-xcffib-0.8.1_1: not found
   [target] python3-devel-3.6.9_1: found (https://alpha.de.repo.voidlinux.org/current)
   [target] libffi-devel-3.2.1_6: found (https://alpha.de.repo.voidlinux.org/current)
   [runtime] python3-cairocffi-1.1.0_1: not found
   [runtime] python3-xcffib-0.8.1_1: not found
=> python3-cairocffi-1.1.0_1: building [python3-module] (dependency of qtile) ...
   [host] python3-setuptools-41.2.0_2: found (https://alpha.de.repo.voidlinux.org/current)
   [host] python3-cffi-1.12.3_1: found (https://alpha.de.repo.voidlinux.org/current)
   [host] python3-xcffib-0.8.1_1: not found
   [target] cairo-devel-1.16.0_1: found (https://alpha.de.repo.voidlinux.org/current)
   [runtime] python3-cffi-1.12.3_1: found (https://alpha.de.repo.voidlinux.org/current)
   [runtime] cairo-1.16.0_1: found (https://alpha.de.repo.voidlinux.org/current)
=> ERROR: python-xcffib: this package cannot be built for x86_64.
```
archs should be noarch and `nocross=yes` on it own line, when I do that it does succeed.

Reply:
Applied.